### DISCUSSION TITLE
Missing entity

### DISCUSSION THREAD

**Discussion by ekkis** [prisma/prisma#13069] [Q&A] [answered]
I have this schema:
```prisma
model Card {
    customer   Customer @relation(fields: [customerId], references: [id])
    customerId Int
    number     String   @unique
    expMon     Int
    expYear    Int
    CVC        Int
}
```
and client code:
```js
import { PrismaClient } from "@prisma/client"
const prisma = new PrismaClient();
var db = prisma["card"]
return await db.findMany({
    where: ...
})
```
however, when I run this I get the following error:

> [user] Error responding: TypeError: Cannot read property 'findMany' of undefined
>     at __WEBPACK_DEFAULT_EXPORT__ (webpack-internal:///(api)/./pages/api/db.js:22:28)
>     at Object.apiResolver (/Users/ekkis/dev/node_modules/next/dist/server/api-utils/node.js:184:15)
>     at processTicksAndRejections (internal/process/task_queues.js:95:5)
>     at async DevServer.runApi (/Users/ekkis/dev/node_modules/next/dist/server/next-server.js:397:9)
>     at async Object.fn (/Users/ekkis/dev/node_modules/next/dist/server/base-server.js:473:37)

and in fact, there is no key "card" in the prisma client.  I've generated that client with the following script in my `package.json`:

```json
  "scripts": {
    "prisma": "PRISMA_CLIENT_ENGINE_TYPE='dataproxy' prisma generate && . .env.local && export DATABASE_URL && prisma db push && prisma db seed",
```
and I've looked in the `node_modules/@prisma/client` folder for clues of the presence of the entity but I don't get how that stuff is built and where I should be looking

can anyone suggest how I might debug/fix this?

**Appendix I**

I found that in `node_modules/.prisma/index.d.ts` actually contains the correct information:
```js
export type Card = {
  customerId: number
  number: string
  expMon: number
  expYear: number
  CVC: number
}
```
so I'm even less sure now why my  code is  failing.  it means I'm regenerating the client correctly

**Comment by ekkis** [comment, marked as answer]
figured it out.  I needed to kill my server as it doesn't pick up the new client until I do

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
